gradle/versions.gradle

```diff
@@ -12,6 +12,9 @@ ext {
     androidMavenGradlePluginVersion = "1.4.1"
 
     // Libraries
+    androidxTestRunnerVersion = "1.2.0"
```

Comment:
Don't forget to update `android-complete` ;)

Reply:
Opened up a PR here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/911